src/main/ui/app/sass/addonindex.scss

```diff
@@ -26,14 +26,32 @@
   float: right;
 
 }
+.maintable{
+margin-bottom:0;
+}
+
+
+.titletype{
 
+margin-right:5%;
+margin-bottom:5px;
+
+}
+.titlename{
+margin-left:-15%;
+margin-bottom:5px;
+
+}
+.mainlist{
+border-width: 0;
+}
 .pushdown {
   margin-top: 40px;
 }
 
 .textdec {
```

Comment:
Textdec is the class used for styling the error codes displayed (I know the name doesn't make sense )